src/AlertDestination/Mail.php

```diff
@@ -82,7 +82,7 @@ private function sendMail(string $to, string $subject, Alert $alert, string $act
     {
         try {
             $message = (new \Swift_Message($subject))
-                ->setFrom(getenv('MAILER_FROM'))
+                ->setFrom($_ENV['MAILER_FROM'])
```

Comment:
This has the same issue as described above.

Reply:
function will never be called if mailer_from is not set